index.js

```diff
@@ -42,6 +42,14 @@ var Dropzone = React.createClass({
     e.stopPropagation();
     e.dataTransfer.dropEffect = 'copy';
 
+    var effectAllowed = e.dataTransfer.effectAllowed;
+
+    // set active drag state only when file is dragged into
+    // (in mozilla when file is dragged effect is "uninitialized")
+    if (effectAllowed !== 'all' && effectAllowed !== 'uninitialized') {
+      return;
```

Comment:
Please don't `return` here, as we need the `this.props.onDragOver(e)` to still be called always. Instead wrap ``` this.setState({
       isDragActive: true
     });``` inside an `if (effectAllowed === 'all' || effectAllowed === 'uninitialized')`